static-build/CMakeLists.txt

```diff
@@ -43,6 +43,14 @@ if (APPLE)
     set(DEPENDENCY_CPPFLAGS "${CMAKE_C_SYSROOT_FLAG} ${CMAKE_OSX_SYSROOT}")
 endif()
 
+option(ENABLE_HARDENING "Enable compiler options that harden against memory corruption attacks" ON)
```

Comment:
I think we should add a changelog entry for this change. We have the `## feature/build` sub-section for build-related changes.

Reply:
Done